ddtrace/contrib/django/middleware.py

```diff
@@ -18,6 +18,27 @@
 log = logging.getLogger(__name__)
 
 
+class TraceExceptionMiddleware(object):
+    """
+    Middleware that traces exceptions raised
+    """
+    def __init__(self, get_response=None):
+        # disable the middleware if the tracer is not enabled
+        # or if the auto instrumentation is disabled
+        self.get_response = get_response
+        if not settings.AUTO_INSTRUMENT:
+            raise MiddlewareNotUsed
+
+    def process_exception(self, request, exception):
```

Comment:
@gabsn can you address this comment so that we avoid to keep the same code in two different places? said that, I think the rest is good!